resources/views/group/view.blade.php

```diff
@@ -40,73 +40,51 @@
               $group_image->image->path;
           }
 
-          $can_edit_group = App\Helpers\Fixometer::hasRole( $user, 'Administrator') || $isCoordinatorForGroup || $is_host_of_group;
-          $can_see_delete = App\Helpers\Fixometer::hasRole( $user, 'Administrator');
+          $can_edit_group = App\Helpers\Fixometer::hasRole($user, 'Administrator') || $isCoordinatorForGroup || $is_host_of_group;
+          $can_see_delete = App\Helpers\Fixometer::hasRole($user, 'Administrator');
           $can_perform_delete = $can_see_delete && $group->canDelete();
 
-          $expanded_events = [];
```

Comment:
Is expanded_events no longer needed?

Reply:
Assignment has been moved to the controller and is still used by the Vue element:
`:events="{{ json_encode($expanded_events, JSON_INVALID_UTF8_IGNORE) }}"`